src/services/cache.ts

```diff
@@ -0,0 +1,14 @@
+import { Service } from "../common";
+
+export class CacheService extends Service {
+  async call(path: string): Promise<unknown> {
+    if (!this.ctx.cache.useCache || !this.ctx.cache.url) {
+      return undefined;
```

Comment:
so if `ctx.cache` is not set, because it hasn't been configured, then this will throw an error. If it is set but `cache.useCache` is `false` or `cache.url` is undefined then it'll just return nothing and not throw. Sound ok?